.github/workflows/nightly_snapshot.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
     cpp_package:
         uses: ./.github/workflows/cpp_package.yaml
         with:
-            extra_cmake_flags: ${{ github.event.inputs.release == 'false' && '-DCPACK_PACKAGE_VERSION=nightly' || '' }}
+            extra_cmake_flags: ${{ github.event.inputs.release != 'true' && '-DCPACK_PACKAGE_VERSION=nightly' || '' }}
```

Comment:
why is `== 'false'` not working and `!= 'true'` works? It might now do the right thing for the nightly, but will fail in release. That's why i prefer failsafe to release.

Reply:
When the build is scheduled via workflow_dispatch, it's indeed `false`, but when the build is scheduled via cron, `github.event.inputs.release` is the empty string.

So this should work for scheduled builds (where it's empty and considered false), as well as for manual release builds where it's true or manual where set to false (see https://github.com/slint-ui/slint/actions/runs/11047149993 and https://github.com/slint-ui/slint/actions/runs/11047164572 for two runs)

Reply:
Ok, the explanation makes sense!